README.md

```diff
@@ -4,198 +4,111 @@ ARIA
 
-In Debian-based systems:
+On Archlinux:
 
-	sudo apt install python-setuptools
-	sudo -H easy_install pip
-	sudo -H pip install virtualenv
-	virtualenv env
+	sudo pacman -S python-setuptools
 
-Or in Archlinux-based systems:
 
-	pacman -S python2 python-setuptools python-pip
-	pip install virtualenv
-	virtualenv env -p $(type -p python2)
+To install ARIA directly from PyPI (using a `wheel`), use:
 
-To install the latest development snapshot of ARIA:
+    pip install aria
 
-	. env/bin/activate
-	pip install git+http://git-wip-us.apache.org/repos/asf/incubator-ariatosca.git
 
-To test it, let's create a service instance from a TOSCA blueprint:
+To install ARIA from source, download the source tarball from [PyPI](https://pypi.python.org/pypi/ariatosca),
+extract it, and then when inside the extracted directory, use:
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml
-	
-You can also get it in JSON or YAML formats:
+    pip install .
+    
+The source package comes along with relevant examples, documentation,
+`requirements.txt` (for installing specifically the frozen dependencies' versions with which ARIA was tested) and more.
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --json
 
-Or get an overview of the relationship graph:
+<br>
+Note that for the `pip install` commands mentioned above, you must use a privileged user, or use virtualenv.
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --graph
 
-You can provide inputs as JSON, overriding default values provided in the blueprint
+Getting Started
+---------------
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --inputs='{"openstack_credential": {"user": "username"}}'
+This section will describe how to run a simple "Hello World" example.
 
-Instead of providing them explicitly, you can also provide them in a file or URL, in either JSON or
-YAML. If you do so, the value must end in ".json" or ".yaml":
+First, provide ARIA with your service-template, naming it `my-service-template`:
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --inputs=blueprints/tosca/node-cellar/inputs.yaml
+	aria service-templates store examples/hello-world/helloworld.yaml my-service-template
+	
+Now create a service based on this service-template, naming it `my-service`:
+	
+	aria services create my-service -t my-service-template
+	
+Finally, start an `install` workflow execution on `my-service` like so:
 
+	aria executions start install -s my-service
 
-CLI
----
+<br>
+You should now have a simple web-server running on your local machine.
+You can try visiting http://localhost:9090 to view your deployed application.
 
-Though ARIA is fully exposed as an API, it also comes with a CLI tool to allow you to work from the
-shell:
+To uninstall and clean your environment, follow these steps:
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml instance
+    aria executions start uninstall -s my-service
```

Comment:
i'd rather not - adding "bash comments" would make this subsection inconsistent with the installation subsection, while making it consistent (i.e. having separate text which explains each and every command) seems like an overkill in this case.